utils/devel/ledgersmb-server-development.psgi

```diff
@@ -16,9 +16,9 @@ package LedgerSMB::FCGI;
 no lib '.';
 
 use FindBin;
-use lib $FindBin::Bin . '/..'; # For our 'old code'-"require"s
-use lib $FindBin::Bin . '/../lib';
-use lib $FindBin::Bin . '/../old/lib';
+use lib $FindBin::Bin . '/../..'; # For our 'old code'-"require"s
+use lib $FindBin::Bin . '/../../lib';
+use lib $FindBin::Bin . '/../../old/lib';
```

Comment:
Why are these here at all? I mean, in the production version we depend on `-I` being passed to the calling process. I think that would be a good idea for the development infrastructure as well.

Reply:
Because we have the same in ledgersmb-server.psgi, only the paths were adjusted to cope with `/bin` and `/utils/devel`.

Reply:
I agree